frontend/src/components/common/Header.tsx

```diff
@@ -8,51 +9,163 @@ import {
+        flexDirection="row"
+        justifyContent="space-between"
+        alignItems="center"
+        display={{ base: "inline", md: "flex" }}
+      >
         <IconButton
           w="24px"
           h="24px"
           flex="1"
           aria-label="menu options"
           onClick={onOpen}
           backgroundColor="transparent"
+          display={{ base: "inline", md: "none" }}
         >
           <MenuIcon />
         </IconButton>
-        <Text pl="21px" flex="9">
-          Community Fridge
-        </Text>
+        <Image
+          objectFit="none"
+          src="header-logo.png"
+          alt="Community Fridge logo"
+          display="inline"
+        />
+        <Stack
+          spacing="2rem"
+          direction="row"
+          display={{ base: "none", md: "flex" }}
+        >
+          <Text>
+            <Link as={ReactLink} to={LANDING_PAGE}>
+              Home
+            </Link>
+          </Text>
+          <Text>
+            <Link as={ReactLink} to={SCHEDULING_PAGE}>
+              Schedule Donation
+            </Link>
+          </Text>
+          {authenticatedUser && (
+            <>
+              <Text>
+                <Link as={ReactLink} to={LANDING_PAGE}>
+                  {" "}
+                  {/* update this to account page */}
+                  My Account
+                </Link>
+              </Text>
+              <Button onClick={onLogOutClick} variant="link" color="black">
+                <Text>Log Out</Text>
+              </Button>
+            </>
+          )}
+          {!authenticatedUser && (
+            <Text>
+              <Link as={ReactLink} to={LOGIN_PAGE}>
+                Sign In
+              </Link>
+            </Text>
+          )}
+        </Stack>
       </Flex>
       <Drawer placement="left" onClose={onClose} isOpen={isOpen} size="xs">
         <DrawerOverlay />
         <DrawerContent backgroundColor="evergreen.100">
           <DrawerBody>
+            <IconButton
+              w="24px"
+              h="24px"
+              float="right"
+              mt="15px"
+              aria-label="close drawer"
+              backgroundColor="transparent"
+              onClick={onClose}
+            >
+              <CloseIcon color="white" />
+            </IconButton>
+            <Image
+              mt="70px"
+              mb="30px"
+              src="drawer-logo.png"
+              alt="Community Fridge logo"
+            />
             <Stack spacing="1rem">
               <Text color="squash.100" textStyle="mobileHeader4">
-                Schedule Dropoff
+                <Link as={ReactLink} to={LANDING_PAGE}>
+                  Home
+                </Link>
               </Text>
               <Text color="squash.100" textStyle="mobileHeader4">
-                <Link as={ReactLink} to={LOGIN_PAGE}>
-                  Sign In
+                <Link as={ReactLink} to={SCHEDULING_PAGE}>
+                  Schedule Donation
                 </Link>
```

Comment:
Since `Link`  is a chakra component, I don't think you need to wrap it with a `<Text/>` component, you can pass in the color and textStyle props directly